torch/_export/passes/replace_sym_size_ops_pass.py

```diff
@@ -10,19 +9,10 @@
 }
 
 
-class _ReplaceSymSizeOpPass(PassBase):
-    """
-    Replace torch.ops.aten.sym_size with torch.ops.aten.sym_size.int
-    and torch.ops.aten.sym_stride with torch.ops.aten.sym_stride.int
-    """
-
-    def call(self, graph_module) -> PassResult:
-        modified = False
-        for module in graph_module.modules():
-            if not isinstance(module, torch.fx.GraphModule):
-                continue
-            for node in module.graph.nodes:
-                if node.target in replacements:
-                    node.target = replacements[node.target]
-                    modified = True
-        return PassResult(graph_module, modified)
+def _replace_sym_size_ops_pass(gm: torch.fx.GraphModule):
+    for module in gm.modules():
```

Comment:
Why do we not do this in top level module?

Reply:
we do run this in the top level module.